AnnoyancesFilter/Cookies/sections/cookies_general.txt

```diff
@@ -1091,6 +1091,7 @@
 ###wt-cookie-box
 ###ww-cookie-information
 ###zaz-nb-plugin-cookie-message
+###iai_cookie
```

Comment:
Better to start with the specific rule.
```adblock
e-baseus.com###iai_cookie
```
Add it to the `cookies_specific` at the begining of the file.